Stripe/STPAPIClient.m

```diff
@@ -129,8 +131,20 @@ - (instancetype)initWithPublishableKey:(NSString *)publishableKey
     return self;
 }
 
+- (void)setApiKey:(NSString *)apiKey {
+    NSURLSessionConfiguration *sessionConfiguration = [NSURLSessionConfiguration defaultSessionConfiguration];
+    NSString *auth = [@"Bearer " stringByAppendingString:apiKey ?: @""];
+    sessionConfiguration.HTTPAdditionalHeaders = @{
+                                                   @"X-Stripe-User-Agent": [self.class stripeUserAgentDetails],
+                                                   @"Stripe-Version": stripeAPIVersion,
+                                                   @"Authorization": auth,
+                                                   };
+    self.urlSession = [NSURLSession sessionWithConfiguration:sessionConfiguration];
```

Comment:
Nevermind, I thought `apiClient.urlSession` was only used for the file upload api but it's more complicated than just that.